### ISSUE TITLE
How to cite eppy?

### ISSUE THREAD

**Issue by airallergy** [santoshphilip/eppy#440]
Not sure if this has been asked before, a quick search did not find anything, how would you like eppy to be cited? eppy is very well deserved to be properly cited in papers and reports, and it would be ideal to have a consistent way to do so. (I came across a few papers citing eppy, but unsurprisingly fairly diverged.)

[Zonodo](https://zenodo.org) seems popular to get a doi for software, as mentioned in [this GitHub doc](https://docs.github.com/en/repositories/archiving-a-github-repository/referencing-and-citing-content). A few alternatives exist I think. Also, GitHub added the function to have [a CITATION file](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-citation-files) in the root at some point.

**Comment by santoshphilip** [comment]
@airallergy , thank you for bringing this to my attention. Citations are not on my radar, since I am not in the academic universe :-). 

the links you have given are very informative. I'll probably pick one of them. I'll bounce my thinking on it off you before I implement.

I do recollect seeing a note on citations for the [CBE Termal comfort tool](https://comfort.cbe.berkeley.edu). It is at the bottom of the page. They have a paper associated with the too. so it is easier to cite.

There are two papers on eppy (I am an author on one of them) that may be worth linking too.

I'll leave this issue open until it comes to a conclusion

**Comment by airallergy** [comment]
I wasn't aware that there are papers on eppy, could you share some links?

If there exists a paper specific about eppy authored by you, I think it is a common practice to cite that, which is the case for the thermal comfort tool you mentioned, also for numpy, SciPy, etc. Otherwise (I found [this](https://publications.ibpsa.org/conference/paper/?id=simbuild2018_C043) for example), I am not sure to be honest.

**Comment by santoshphilip** [comment]
Yes that is the [link](https://publications.ibpsa.org/conference/paper/?id=simbuild2018_C043) of the paper on eppy as a tool.

eppy was partly funded by a project that @jasonglazer did for ASHRAE. He published one or two papers on the project. 

Jason sent me some links  long ago on other papers that mention eppy:

> Paper referencing Eppy
> 
> I just came across this
> 
> https://www.sciencedirect.com/science/article/abs/pii/S0306261921007492?via%3Dihub
> 
> Maybe it is not that uncommon
> 
> https://scholar.google.com/scholar?hl=en&as_sdt=0%2C24&q=energyplus+eppy&btnG=
> 
> Jason
>

**Comment by JasonGlazer** [comment]
The paper I wrote as part of the ASHRAE 1651-RP project related to eppy was:

https://publications.ibpsa.org/conference/paper/?id=simbuild2016_C030

In it I cited eppy by using:

![Capture2](https://github.com/santoshphilip/eppy/assets/2582150/a8a9a643-fc3f-48eb-8ed8-7d99d7aa20f1)

**Comment by airallergy** [comment]
Citing these existing papers mentioning eppy now seems less ideal, as they cited eppy themselves, one way or another :)

I suppose it boils down to two options for a standardised/recommended eppy citation:

1. write a manual entry for the CITATION file
2. create a Zonodo, or alike, entry and include it in the CITATION file

**Comment by santoshphilip** [comment]
@airallergy , I agree. I'll just pick one of them and go with it. Will close this issue once that happens

**Comment by airallergy** [comment]
Happy new year @santoshphilip! Wondering if there has been any update on this?

**Comment by santoshphilip** [comment]
Happy new year. This issue slipped off my radar. I think I should riveiw it once more and pick an option to go with.
